docs/csharp/programming-guide/concepts/linq/creating-xml-trees-linq-to-xml-2.md

```diff
@@ -212,6 +212,6 @@ Console.WriteLine("Child2 was {0}",
 //    Child2 was attached  
 ```
 
-## See also
+## See Also
```

Comment:
actually, this one has the right casing. Our headings are supposed to be sentence case. See #808
We've been slowly changing to the right one over time.

Reply:
I was wondering and took the most used one... wrong choice 👎 
I'll apply the correct casing in my next commit.